### ISSUE TITLE
Don't always hear "go beyond-edge" responses 

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#332]
The "go beyond edge" responses were verified for keyboard interaction in #247.

However, yesterday while testing with iOS 14.3 and VO on iPhone7, I was not getting responses when the right hand tried to go beyond the top. I did, however, hear the response for the left hand, when trying to go lower than "near bottom". 

Today, however, I am having trouble double verifying the behaviour, likely due to changes in https://github.com/phetsims/scenery/issues/1094

**Comment by zepumph** [comment]
I didn't know if this capability was supported on iOS. That said, it is possible that https://github.com/phetsims/scenery/issues/1094 just changed the behavior from what you experienced 2 days ago. 

When I am at the border, and I swipe towards the border, I only get the object response. I also tested this behavior in GFLB and got the same behavior on master, as well as with https://phet.colorado.edu/sims/html/gravity-force-lab-basics/latest/gravity-force-lab-basics_en.html.

Basically I thought that the screen reader doesn't provide events to the browser in this case, it just repeats the object response. I could be wrong though. Do you know if we have gotten these on iOS before?

**Comment by terracoda** [comment]
Ratio has to be locked to get this response.

**On Mac OS no issue**
I have verified that the "go beyond edge" responses fire for me when using the keyboard or the mouse on MacOS 10.13.6 (sorry I haven't updated). They fire when using either hand slider, and using either WS keys or Up/Down Arrow keys on the Both Hands interaction. The responses work with mouse and keyboard.

There is something inconsistent when using iOS. Here's my summary.

**With VO and Safari on iOS 14.3 (iPhone7)**
- With Numbered Tick Marks selected 
    - I always get the beyond-top response ("Right hand at top. Move hands down to explore more.") when I try to go beyond Tick mark 10. 
    - However, I never get the beyond-bottom response ("Left hand near bottom. Move hands up to explore more.") when I try to go beyond the lowest possible locked ratio position ("both hands near tick mark zero").
- With No Tick Marks 
    -  I never get the beyond-top or beyond-bottom responses 
- With "plain" Tick Marks 
    -  I never get the beyond-top or beyond-bottom responses
 
I have verified that it is possible for the response to fire on iOS (using VO swipes) because the beyond-top response fires consistently with Numbered Tick Marks selected.

@zepumph, there seems to be something different going on for the beyond-bottom response when Numbered Tick Marks is selected, and when the other 2 views are selected.

Perhaps, you could verify how it is working for the one case, the beyond-top responses when Numbered Tick Marks is selected?

**Comment by zepumph** [comment]
@terracoda and I discussed this together today. We found https://github.com/phetsims/gravity-force-lab-basics/issues/185#issuecomment-545180225 which was this exact question that we discussed when publishing GFLB. Over there @jessegreenberg and I found that VO doesn't give the browser any events when moving "beyond" the min/max of a slider that is currently at that min or max. 

Ratio and Proportion is specifically strange when the ratio is locked. This is because you can trigger the "go beyond" alert by moving the "other" hand, which isn't at the mix or max. For example, if the target challenge is 1:2, ratio is locked, and the right hand is at max. In that case moving the left hand up will have VO send an event to the browser to try to move the slider from 5 to 5.5. Then the RaP model will compute it back to 5 to constrain the ratio in range and keep it locked. Because the browser gets input, we will also get the "go beyond" edge alert. 

@terracoda and I are happy to have cracked the case on this weirdness, and don't feel like there is anything else to do here. Again I'll tag this as a `wontfix`.
